src/sentry/integrations/jira/issue_hook.py

```diff
@@ -48,7 +50,8 @@ def get(self, request, issue_key, *args, **kwargs):
                 if not group_link:
                     raise GroupLink.DoesNotExist()
                 group = Group.objects.get(id=group_link.group_id)
-            except (ExternalIssue.DoesNotExist, GroupLink.DoesNotExist, Group.DoesNotExist):
+            except (ExternalIssue.DoesNotExist, GroupLink.DoesNotExist, Group.DoesNotExist) as e:
+                scope.set_tag("failure", e)
```

Comment:
@ceorourke how does this work if the error is an object? Does it just call the `__str__` method on the error?

Reply:
oop maybe I should do a e.__class__.__name__ instead

Reply:
well that formatted weird but e dunder class dunder name

Reply:
hahah yea I know what you mean, I think that makes sense